tests/km_core_tests.bats

```diff
@@ -1860,8 +1868,8 @@ EOF
    run touch $WORKDIR/continue
    assert_success
```

Comment:
Not sure what we are checking here - can the test affect this touch somehow?

Reply:
If touch fails the test will hang.
I'd like to fail the test rather that leave it hanging for 250 seconds and then have it timeout.